packages/f/freefilesync/package.yml

```diff
@@ -1,38 +1,49 @@
 name       : freefilesync
-version    : 10.3
-release    : 4
+version    : '13.6'
+release    : 5
 source     :
-    - https://freefilesync.org/download/FreeFileSync_10.3_Source.zip : 9e958ae968d4e8aff40e547edc575735da73fe0dd9d2558b060f11471a24a0ef
+    - https://sources.getsol.us/FreeFileSync_13.6_Source.zip : 949e0b8a2b14cbcc3a38f778e3026e7c7fa89756308ce7c541fab38f71d66f6c
+homepage   : https://freefilesync.org/
 license    : GPL-3.0-or-later
 component  : network.util
 summary    : Free backup software to synchronize files and folders
 description: |
     FreeFileSync is a folder comparison and synchronization software that creates and manages backup copies of all your important files.
 builddeps  :
-    - pkgconfig(gtk+-2.0)
+    - pkgconfig(gtk+-3.0)
+    - pkgconfig(libcurl)
     - pkgconfig(libnotify)
+    - pkgconfig(libssh2)
     - pkgconfig(sm)
     - pkgconfig(xxf86vm)
     - libboost-devel
-    - wxwidgets30-devel
+    - wxwidgets-devel
 setup      : |
     pushd ..
-    %apply_patches
+    %patch -p1 -i $pkgfiles/pkg-config.patch
+    %patch -p1 -i $pkgfiles/wx-config-version.patch
+    %patch -p1 -i $pkgfiles/ffs_devuan.patch
+    %patch -p1 -i $pkgfiles/remove_upstream_build_optimization.patch
+    %patch -p1 -i $pkgfiles/ffs_devuan_gtk3.patch
+    %patch -p1 -i $pkgfiles/zlib-dep.patch
+    %patch -p1 -i $pkgfiles/ffs_no_check_updates.patch
+    %patch -p1 -i $pkgfiles/ffs_sftp.patch
+    %patch -p1 -i $pkgfiles/skip-missing-Animal.patch
+    %patch -p1 -i $pkgfiles/ffs_icon_loader.patch
+    %patch -p1 -i $pkgfiles/disable_wxWidgets_uncaught_exception_handling.patch
     popd
 build      : |
-    %make -C Source
-    %make -C Source/RealTimeSync
+    %make -C Source exeName=FreeFileSync
+    %make -C Source/RealTimeSync exeName=RealTimeSync
 install    : |
-    %make_install -C Source
-    %make_install -C Source/RealTimeSync
 
+    install -Dm00755 $workdir/Build/Bin/FreeFileSync $installdir/usr/bin/FreeFileSync
+    install -Dm00755 $workdir/Build/Bin/RealTimeSync $installdir/usr/bin/RealTimeSync
     install -Dm00644 $pkgfiles/FreeFileSync.desktop $installdir/usr/share/applications/FreeFileSync.desktop
     install -Dm00644 $pkgfiles/RealTimeSync.desktop $installdir/usr/share/applications/RealTimeSync.desktop
 
-    unzip Build/Resources.zip FreeFileSync.png RealTimeSync.png
-    install -Dm00644 FreeFileSync.png $installdir/usr/share/pixmaps/FreeFileSync.png
-    install -Dm00644 RealTimeSync.png $installdir/usr/share/pixmaps/RealTimeSync.png
-
-    # Set file permissions
-    find $installdir/usr/share/FreeFileSync -type f -print0 | xargs -0 chmod 644
-    find $installdir/usr/share/doc/FreeFileSync -type f -print0 | xargs -0 chmod 644
+    #unzip Build/Resources.zip FreeFileSync.png RealTimeSync.png
```

Comment:
Should this commented out line be removed?

Reply:
I thought I removed it, thanks for letting me know